stb_vorbis.c

```diff
@@ -553,7 +553,7 @@ enum STBVorbisError
 #include <string.h>
 #include <assert.h>
 #include <math.h>
-#if !(defined(__APPLE__) || defined(MACOSX) || defined(macintosh) || defined(Macintosh))
```

Comment:
I don't have that list, though, so I would just have to break everyone and wait for reports to come in.

But maybe these days it works everywhere.